CorsixTH/Src/iso_fs.cpp

```diff
@@ -133,63 +283,34 @@ bool iso_filesystem::initialise(FILE* fRawFile)
 
     // The first 16 sectors are reserved for bootable media.
     // Volume descriptor records follow this, with one record per sector.
-    for(uint32_t iSector = 16; seek_to_sector(iSector); ++iSector)
-    {
-        uint8_t aBuffer[190];
-        if(!read_data(sizeof(aBuffer), aBuffer))
+    for(uint32_t iSector = first_filesystem_sector; seek_to_sector(iSector); ++iSector) {
+        uint8_t aBuffer[root_directory_offset + root_directory_entry_size];
+        if(!read_data(sizeof(aBuffer), aBuffer)) {
             break;
+        }
         // CD001 is a standard identifier, \x01 is a version number
-        if(std::memcmp(aBuffer + 1, "CD001\x01", 6) == 0)
-        {
-            if(aBuffer[0] == vdt_privary_volume)
-            {
-                sector_size = read_native_int<uint16_t>(aBuffer + 128);
-                find_hosp_directory(aBuffer + 156, 34, 0);
-                if(file_count == 0)
-                {
+        if(std::memcmp(aBuffer + 1, "CD001\x01", 6) == 0) {
+            if(aBuffer[0] == vdt_primary_volume) {
```

Comment:
At other places you change it to `if (...`, ie with a space behind the keyword. You may want to look for `if(` and a the usual others (`for(` and `while(`). Wiki code style probably also needs adapting then.

Reply:
Not a big fan that I missed these, but I will do a format patch later (soon) to fix the C++ format consistency project wide. Skip for now.